client/dashboard/data/site-staging-site.ts

```diff
@@ -7,3 +7,10 @@ export async function deleteStagingSite( stagingSiteId: number, productionSiteId
 		apiNamespace: 'wpcom/v2',
 	} );
 }
+
+export async function getAutomatedTransferStatus( siteId: number ) {
+	return wpcom.req.get( {
+		path: `/sites/${ siteId }/automated-transfers/status`,
```

Comment:
So it turns out I am implementing a similar thing here:

- https://github.com/Automattic/wp-calypso/pull/104493

But I'm using a different endpoint... `/sites/:id/atomic/transfers/latest`. I'm not sure the difference between the endpoints -- it seems we use both in Calypso 😅 maybe mine is a more recent endpoint? Can we check?

It would be better if we can just use one version of AT-related endpoints.